obj_gen.cpp

```diff
@@ -529,10 +529,10 @@ imported_keylist::imported_keylist(const char *filename)
 
 imported_keylist::~imported_keylist()
 {
-    while (!m_keys.empty()) {
-        free(m_keys.front());
-        m_keys.erase(m_keys.begin());
+    for (key* k : m_keys) {
```

Comment:
Let's stick to the old style for old compiler compatibility.

Reply:
Sure. I just changed the loop style.